### PR TITLE
Support/wagtail 52

### PR DESCRIPTION
**Pull request by katdom13**
### Changes

- Update test matrices
- Drop tests for Wagtail 4.2 and 5.0 as they have reached EOL

### CODE REVIEW

File: tox.ini
Comment (katdom13) [inline tox.ini]: I had to also drop the `wagtailmain` tests because the `modeladmin` is already deprecated on the newest wagtail version.

Making a note here to put the `wagtailmain` tests back for Wagtail 6.0+ upgrades.

File: tox.ini
Comment (nickmoreton) [inline tox.ini]: I think here the matrix needs to be broken into 2 sections as Wagtail 4.1 doesn't work with django 4.2. wagtail 5.0 is the first version that supported Django 4.2
It looks like why the tests are failing.

**Review by nickmoreton — CHANGES_REQUESTED** [review]

File: tox.ini
Comment (nickmoreton) [reply tox.ini]: Also the merge to main for the 5.1 upgrade didn't run OK.
I think this might have been because of the djangomain issue and we should fix up main first then rebase 5.2 to pick up stopping djangomain for now.

Comment (katdom13) [reply tox.ini]: Hi @nickmoreton ,
This is a fix PR that should be pushed to the `main` branch. Could you please approve this one?
https://github.com/torchbox-forks/wagtail-rangefilter/pull/5

This PR builds on that.

Thanks!

**Review by nickmoreton — APPROVED** [review]
Hi @katdom13 This looks good to go once the extra PR has been merged. Thanks

**Pull request merged**
